.github/workflows/ci.yml

```diff
@@ -10,35 +10,48 @@ jobs:
     runs-on: ubuntu-latest
     steps:
       - uses: actions/checkout@v4
-      - run: cp wrangler.toml.example wrangler.toml
-      - uses: actions/setup-node@v4
+
+      - name: Install Node.js
+        uses: actions/setup-node@v4
         with:
           node-version: 22
-      - run: npm ci
+
+      - name: Install dependencies
+        run: npm ci
+
+      - name: Install Wrangler
+        run: npm install -g wrangler
+
```

Comment:
```suggestion
```

It should work without this as well, as long as we have `wrangler` in `devDependencies`